File: class-and-object/Fruit1.java
/**
 * Fruit
 */
public class Fruit1 {

    String nama;
    String warna;
    int harga;

    public void warnaBuah(String warnaBuah) {
        warna = warnaBuah;
    }

    public void hargaBuah(int hargaBuah) {
        harga = hargaBuah;
    }

    public void namaBuah(String namaBuah) {
        nama = namaBuah;
    }

    public void detailBuah() {
        
        System.out.println("Nama buah : " + nama);
        System.out.println("Warna : " + warna);
        System.out.println("Harga buah : " + harga);

    }
    
}

File: class-and-object/fruit1.py
class Fruit1:

    def warnaBuah(warnaBuah):
        global warna
        warna = warnaBuah

    def namaBuah(namaBuah):
        global nama
        nama = namaBuah

    def hargaBuah(hargaBuah):
        global harga
        harga = hargaBuah

    def detailBuah():
        print("Nama buah : " + nama)
        print("Warna : " + warna)
        print("Harga buah : " + str(harga))
